### PR TITLE
fix using spy

### PR DESCRIPTION
**Pull request by victoralfaro-dotcms**

### CODE REVIEW

File: dotCMS/src/test/java/v2/languages/LanguagesResourceTest.java [hunk 1]
```diff
@@ -1,12 +1,12 @@
 package v2.languages;
 
 import static org.junit.Assert.assertEquals;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```
Comment (dev-dotcms) [inline dotCMS/src/test/java/v2/languages/LanguagesResourceTest.java:4]: Codacy found an issue: [Avoid unused imports such as 'org.mockito.Mockito'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5971966)

File: dotCMS/src/test/java/v2/languages/LanguagesResourceTest.java [hunk 2]
```diff
@@ -38,8 +40,12 @@ public void list() throws DotSecurityException, DotDataException {
         when(initDataObject.getUser()).thenReturn(user);
         when(webResource.init(request, httpServletResponse,true)).thenReturn(initDataObject);
         when(languageAPI.getLanguages()).thenReturn(languages);
+        final LanguageView languageView = mock(LanguageView.class);
 
-        final LanguagesResource languagesResource = new LanguagesResource(languageAPI, webResource);
+        final LanguagesResource languagesResource = spy(new LanguagesResource(languageAPI, webResource));
+        Mockito.doReturn((Function<Language, LanguageView>) language -> languageView)
```
Comment (dev-dotcms) [inline dotCMS/src/test/java/v2/languages/LanguagesResourceTest.java:46]: Codacy found an issue: [Unnecessary use of fully qualified name 'Mockito.doReturn' due to existing static import 'org.mockito.Mockito.*'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5971966)

File: dotCMS/src/test/java/v2/languages/LanguagesResourceTest.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/test/java/v2/languages/LanguagesResourceTest.java:4]: Codacy found an issue: [Avoid unused imports such as 'org.mockito.Mockito'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5971966)

File: dotCMS/src/test/java/v2/languages/LanguagesResourceTest.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/test/java/v2/languages/LanguagesResourceTest.java:46]: Codacy found an issue: [Unnecessary use of fully qualified name 'Mockito.doReturn' due to existing static import 'org.mockito.Mockito.*'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5971966)

**Review by erickgonzalez — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

**Pull request merged**